x/metadata/keeper/scope.go

```diff
@@ -528,7 +554,14 @@ func (k Keeper) ValidateUpdateScopeOwners(
 		if err := validateRolesPresent(proposed.Owners, scopeSpec.PartiesInvolved); err != nil {
```

Comment:
I just noticed that lines 537-542 are equivalent to 554-559. Are we able to pull them out of the branch?

Reply:
Good call. I pulled them out here and also up in ValidateWriteScope.